flambe/experiment/experiment.py

```diff
@@ -274,8 +274,12 @@ def run(self, force: bool = False, verbose: bool = False, debug: bool = False, *
 
         # By default use all CPUs if no GPU is present
         devices = self.devices if self.devices else None
-        if devices is None and utils.local_has_gpu():
-            devices = {"cpu": 4, "gpu": 1}
+        if devices is None:
+            cluster_devices = ray.cluster_resources()
+            if 'GPU' in cluster_devices or 'gpu' in cluster_devices:
```

Comment:
nit for avoiding repeating the check:
```
if any(x in cluster_devices for x in ['GPU', 'gpu']):
```
or
```
if any(map(lambda x: x in cluster_devices, ['GPU', 'gpu'])):
```

Reply:
I actually find that harder to read TBH. I would do it if there were more than 2, but here I feel like it's okay